### DISCUSSION TITLE
Windows time gets misconfigured when correct time is configured in chromeos and vice-versa

### DISCUSSION THREAD

**Discussion by omansh-krishn** [sebanc/brunch#1646] [General]
After installation, my windows time was changed, but when i corrected it, it got changed in chromeos.
Same happened when i installed arch, debian and other linux distributions, but there was a command that can get rid of it. I doesn't know how to fix that in chromeos.

**Comment by shenqingyi9** [comment]
Why not switch your Windows to use UTC time?
```bat
reg add "HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\TimeZoneInformation" /v RealTimeIsUniversal /d 1 /t REG_DWORD /f
```
